src/wp-includes/class-json.php

```diff
@@ -774,7 +816,7 @@ function decode($str)
                                 $parts = array();
 
                                if (preg_match('/^\s*(["\'].*[^\\\]["\'])\s*:/Uis', $slice, $parts)) {
- 	                              // "name":value pair
+                                    // "name":value pair
```

Comment:
Don't change the whitespace here:

```suggestion
 	                              // "name":value pair
```